### ISSUE TITLE
Setup Datadog User for RDS Monitoring

### ISSUE THREAD

**Issue by msnwatson** [department-of-veterans-affairs/abd-vro#2504]
**User Story**

As a VRO platform engineer, I want clear a path for implementing RDS metrics monitoring, so that we can get a better sense of RDS DB instance status.

**Acceptance Criteria**

1. Datadog user for the database has been created for all of our environments
2. Document the creation of this datadog user as an initialization step for RDS dbinstances in the future.
3. Create a dashboard displaying the database metrics being reported in datadog. Metrics below: 
4. Copied from #2138 : "We anticipate that follow-on tickets from this story will include: Document what we do when we start running low on storage - what will the plan be?" This ticket should be created as part of AC for this ticket.
5. Metrics we need to monitor: 1. What have we allocated for compute capacity 2. How much are we using? 3. Storage allocation capacity 4. How much are we using? 

**Notes about work**
- You'll want to use these docs as guidance: https://github.com/department-of-veterans-affairs/lighthouse-di-documentation/blob/main/docs/platform-tools/datadog.md#rds-metrics-collection-in-datadog
- Along with our own documentation for guidance at https://github.com/department-of-veterans-affairs/abd-vro/wiki/VRO-Database
- This is where the work stopped: https://dsva.slack.com/archives/C03UA9MV1EH/p1713221148732019
**-  Most recent update:** 
-  This is where we are in terms of metrics: https://lighthouseva.slack.com/archives/C03UA9MV1EH/p1715000405730929
             -  Per the team's discussion, we have agreed to close this ticket and create a new ticket to follow up after LHDI 
                 fixes this.

**Comment by ponny-m** [comment]
Still blocked since waiting for LHDI to respond to this [message](https://lighthouseva.slack.com/archives/C03UA9MV1EH/p1712593582211689) from Tyler

**Comment by meganhicks** [comment]
When the tenant tries grant SELECT ON pg_stat_database to datadog they receive: WARNING:  no privileges were granted for "pg_stat_database"
Tenant has confirmed they tried admin and superuser but resulted in the same error.

**Comment by meganhicks** [comment]
Open Questions:

1. Datadog user for the database has been created for all of our environments- this is done right?
2. Document the creation of this datadog user as an initialization step for RDS dbinstances in the future. done too?
3. Create a dashboard displaying the database metrics being reported in datadog. How did we determine what we would track? Are we aiming for what is out of the box or only looking for usage and allocation metrics for VRO's implementation of RDS?

**Comment by meganhicks** [comment]
@BerniXiongA6 
Initially, we were informed that LHDI couldn't enable monitoring for our main DB, RDS. This left us puzzled and appeared to be a product oversight. Considering this is our primary database housing a significant amount of veteran claim information, it's crucial for us to monitor it and establish a retention policy. We've diligently followed all instructions provided: setting up a user with monitoring permissions and working with an engineer to configure metrics. However, we've encountered a recent roadblock due to a missing feature, resulting in a lack of monitoring capabilities. Despite requesting a call to discuss the timeline, Tyler mentioned it wasn't necessary and assured us he's addressing the issue. We urgently need clarity on the timeline. Furthermore, we've repeatedly asked for improved Datadog documentation, as the current resources only provide limited assistance.

**Comment by meganhicks** [comment]
now unblocked -https://dsva.slack.com/archives/C03UA9MV1EH/p1713481463133529?thread_ts=1713193534.639459&cid=C03UA9MV1EH
